org.gnome.frogr.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "org.gnome.frogr",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "frogr",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Needed for frogr to access your pictures */
```

Comment:
I've tried something like that a while ago, and I even had `--filesystem=home` for a while, but that was very inconvenient in common use cases that I'd like frogr to support, such as loading pictures from an external drive (or camera) or from a remote server (e.g. NAS).

I understand, though, that `host` is a much wider permission than any of those, so I'm open to change it if that makes sense for flathub (after all, people can override those with `flatpak override`)

What do you think?